src/patternfly/components/Masthead/masthead-brand.hbs

```diff
@@ -0,0 +1,15 @@
+<{{#if masthead-brand--type}}{{masthead-brand--type}}{{else}}a{{/if}} class="pf-c-masthead__brand{{#if masthead-brand--modifier}} {{masthead-brand--modifier}}{{/if}}"
+  {{#if masthead-brand--attribute}}
```

Comment:
nit - typically we put `--attribute` as the last thing in the inline attrs, so that whatever is passed there would override anything else the template might add. So the `--attribute` block would come after `--href` block.